cmd/cli/config/config.go

```diff
@@ -103,6 +103,11 @@ func getConfigPath(p ...string) (string, error) {
 		}
 	}
 
-	cfgPath = path.Join(cfgPath, configName)
-	return cfgPath, nil
+	switch {
+	case util.FileExists(cfgPath) == nil:
+		return cfgPath, nil
+	case util.DirectoryExists(cfgPath) == nil:
+		return path.Join(cfgPath, configName), nil
+	}
+	return "", fmt.Errorf("No such file or directory: %s", cfgPath)
```

Comment:
chained errors should be lowercased according to [Golang's wiki](https://github.com/golang/go/wiki/Errors)